app/gui/src/dashboard/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -22,14 +20,10 @@ export const BREADCRUMBS_STYLES = tv({
   slots: { separator: 'text-primary last:hidden w-2.5 h-2.5 mt-[0.5px]' },
 })
 
-/**
- * The type of the `onDrop` callback.
- */
+/** The type of the `onDrop` callback. */
```

Comment:
I'm not sure this makes sense, collapsing docs makes it harder to edit them, and creates extra noise in PRs.

Reply:
For me, the "collapsed" creates actually less noise - it makes simple and small things looking simple and small, without sacrifying docs.

Reply:
^ yeah i am mostly doing this because this was suggested before and the reasoning sounds solid enough

Reply:
@farmaazon @somebody1234 don't you guys mind adding this plugin then? https://github.com/hosseinmd/prettier-plugin-jsdoc

Reply:
sounds good to me! big fan of auto formatting

Reply:
> @farmaazon @somebody1234 maybe we can add this plugin? https://github.com/hosseinmd/prettier-plugin-jsdoc

Can it be synchronized with our eslint config? (Or other way round?)

Reply:
@farmaazon do we do any formatting for jsdoc using eslint?

Reply:
Anyway, it seems like we can granularly adjust the prettier plugin or disable conflicting rules in eslint

Reply:
Created an issue, please let's move the discussion there: https://github.com/enso-org/cloud-v2/issues/1782

Reply:
> formatting for jsdoc in eslint?
not 100% sure but i don't think so, we do have jsdoc rule that mandate adding comments but i don't believe there are restrictions on formatting

i *think* we do have `eslint-plugin-prettier` though but that would of course not conflict with prettier